src/CLI/clitree/cli-xml/sonic_types.xml

```diff
@@ -41,6 +41,20 @@ limitations under the License.
         help=""
         />
     <!--=======================================================-->
+    <PTYPE
+        name="LAG_ID"
+        method="integer"
+        pattern="0..9999"
```

Comment:
Does it have to be 1..128? Please confirm.

Reply:
I initially had it 1..128, but then I found out on Sonic NOS the allowed range is '<0-9999>'